src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1196,6 +1249,12 @@ namespace winrt::TerminalApp::implementation
         eventArgs.Cancel(true);
     }
 
+    void TerminalPage::_CloseWarningPrimaryButtonOnClick(Windows::UI::Xaml::Controls::ContentDialog sender,
+                                                         Windows::UI::Xaml::Controls::ContentDialogButtonClickEventArgs eventArgs)
```

Comment:
Unused parameters could have the variable names removed or commented out.

Reply:
commented out